src/main/java/cnlabs/CustomerWebsite/Services/BookService.java

```diff
@@ -0,0 +1,45 @@
+package cnlabs.CustomerWebsite.Services;
+
+import cnlabs.CustomerWebsite.Models.Book;
+import cnlabs.CustomerWebsite.Repos.BookRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class BookService {
+    @Autowired
+    final BookRepository bookRepository;
+
+    public List<Book> getAllBooks() {
+        return bookRepository.findAll();
+    }
+
+    public List<Book> getAvailableBooks() {
+        return bookRepository.findAll().stream().filter(book -> book.getCustomer() == null).collect(Collectors.toList());
```

Comment:
For readability, it is common to format each subsequent operation on a new line, like:

```suggestion
        return bookRepository.findAll().stream()
            .filter(book -> book.getCustomer() == null)
            .collect(Collectors.toList());
```